README.md

```diff
@@ -176,6 +176,21 @@ manually before the backward compatibility is dropped. For every configuration u
 [the default agent config](https://github.com/signalfx/splunk-otel-collector/blob/main/cmd/otelcol/config/collector/agent_config.yaml)
 as a reference.
 
+### From 0.114.0 to 0.115.0
+
+- Replace the sapm exporter in the traces pipline with otlphttp.
+  
+  More details: https://github.com/signalfx/splunk-otel-collector/pull/5588.
```

Comment:
Also, mention that `samp` exporter is still working as before, it's just deprecated. We need to make it sound that it's not super critical to update right away.